src/services/account/docs/index.js

```diff
@@ -0,0 +1,74 @@
+module.exports = {
+	accountService: {
+		description: "A service for accounts",
+		find: {
+			summary: "find"
+		},
+		create: {
+			summary: "create"
+		},
+		remove: {
+			summary: "remove"
+		},
+		update: {
+			summary: "update"
+		},
+		patch: {
+			summary: "patch"
+		},
+		get: {
+			summary: "get"
+		},
+		CustomJWTService: {
+			create: {
+				parameters: [
+					{
+						description: "Userdata for the User to be created",
+						required: true,
+						name: "Userdata",
+						type: "data"
+					}
+				],
+				summary: "Creates a new Account"
```

Comment:
does it really creates a new account here? I think the `CustomJWTService` just creates a valid json webtoken for the corresponding user ...